# 2017-01-09: Siva Prasad Meduri <siva.meduri@nxp.com>
# Patch to fix disutils path adjustment 

--- a/Lib/distutils/command/build_scripts.py
+++ b/Lib/distutils/command/build_scripts.py
@@ -89,6 +89,7 @@ class build_scripts (Command):
                     adjust = 1
                     post_interp = match.group(1) or ''
 
+            adjust = 0
             if adjust:
                 log.info("copying and adjusting %s -> %s", script,
                          self.build_dir)
